graphics/mupdf/Portfile

```diff
@@ -1,14 +1,15 @@
 # -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
 
 PortSystem          1.0
+PortGroup           compiler_blacklist_versions 1.0
 PortGroup           openssl 1.0
 
 name                mupdf
 
 # Keep synchronized with port:py-mupdf.
 version             1.23.4
 
-revision            0
+revision            1
```

Comment:
no "revision" increase since you are only fixing a build issue. 
However, setting the correct standard should pick an appropriate compiler so I am not sure why this is required. I am not sure if the order is important but if so, is just moving the original statement before the "build.args" sufficient?

Reply:
I was not aware that specifying the required C++ standard would cause an appropriate compiler to be picked, but it makes sense. Simply moving the statement up did the trick. Regarding the order, I guess it is important because `build.args` depends on `configure.cxx`, which is not set until `compiler.cxx_standard` statement is processed?

To avoid cluttering the Portfile when moving `compiler.cxx_standard` up, I moved the other statement in the block (`use_parallel_build`) up as well (and put it next to `use_configure`). I will update the PR in a minute.

Reply:
According to the docs, the PR would be retargeted automatically if I renamed the branch. Instead it was automatically closed. I will create a new PR instead.